src/Generator/GumbelBarnettGenerator.jl

```diff
@@ -148,4 +149,49 @@ function τ⁻¹(::Type{T}, tau) where T<:GumbelBarnettGenerator
     end
     # Use the bisection method to find the root
     return Roots.find_zero(θ -> _gumbelbarnett_tau(θ) - tau, (0.0, 1.0))
+end
+
+# Edge utilities and robust bracketing
+_GB_EPSA = 1e-12                 # separa de 0
+_GB_EPSB = 1e-12                 # separa de 1
+c_GB_TOLV = 1e-12                 # tolerancia en valor
+
+# Internal grids to rescue bracketing if there are numerical problems
+_GB_GRID_A = (1e-12, 1e-10, 1e-8, 1e-6, 1e-4, 1e-3, 5e-3, 1e-2, 5e-2)
+_GB_GRID_B = (1 - 1e-12, 1 - 1e-10, 1 - 1e-8, 1 - 1e-6, 1 - 1e-4, 0.999, 0.99, 0.95, 0.9)
+function ρs_GB(θ::Real)

```

Comment:
Again, clarify what happens when d > 2. Also for the \tau and \tau_inv functions, same issue !!!